### DISCUSSION TITLE
Client not loaded in PXE on second VM.

### DISCUSSION THREAD

**Discussion by Sistem-Pack** [ltsp/ltsp#802] [Q&A] [answered]
Good time of day, community!
Can someone help me in my trouble? 
I use two VM. First VM is LTSP-host and second VM is a client.
I don't understand how to configure dnsmasq for my second VM load as LTSP-client.
If I use VMware Workstation NAT on VM-Network adapter for my VM's, own DHCP has set ip-address to my second VM(LTSP-client) and loading will be successfull.
But I can create some network adapter and set static ip-address to VM(LTSP-host) and configure dnsmasq, then my second VM(LTSP-client) is not loaded, and I don't know why.

**Comment by alkisg** [comment, marked as answer]
Hello,

VirtualBox or Qemu/KVM/virt-manager are recommended instead of VMWare, because they're open source, and more people are using them, and more people can give instructions about them.

Personally the last time I've seen VMWare was like 20 years ago, so I'll give instructdions for VirtualBox and hope that VMWare is similar.

First method is to use bridged networking for both VMs. In ltsp-server, the `ltsp dnsmasq` command will configure it in proxyDHCP mode. So your main DHCP server will give the IPs, the ltsp-server will give the boot filename only, and the ltsp-client will be able to boot.

Second method is to use some kind of "internal network". Configure the ltsp-server VM with two NICs. Attach the first NIC to NAT and configurure a dynamic/DHCP IP. This will be the "Internet connection". Attach the second NIC to "internal network" and give it a static IP of 192.168.67.1. Then run `ltsp dnsmasq`. This will configure the ltsp-server as the DHCP server for the "internal network".
Then create the ltsp-client VM and attach it to the "internal network". It will get a 192.168.67.x IP from the ltsp-server.
